lib/generate/patient_imports.rb

```diff
@@ -105,12 +105,14 @@ def write_cohort_import_csv
           ]
         end
       end
+      cohort_import_csv_filepath.to_s
     end
 
     def write_class_import_csv
+      # stree-ignore
       patients
         .group_by(&:school)
-        .each do |school, school_patients|
+        .map { |school, school_patients|
```

Comment:
```suggestion
        .filter_map { |school, school_patients|
```

This is equivalent to `.map.compact`.